roles/k8s-ut/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+- name: Install prereq packages
+  yum:
+    name: "{{ packages }}"
+    state: present
+  vars:
+    packages:
+      - git
+      - make
+      - gcc
+
+- name: Create a shell script
+  copy:
+    content: |
+      #!/bin/bash
+      set -o errexit
+
+      echo 'This is a script to run k8s UT tests'
+      useradd nonroot
+      su - nonroot <<EOF
+      set -o errexit
+
+      git clone https://github.com/kubernetes/kubernetes
```

Comment:
TODO: explore how to checkout a particular commit instead of master.

Reply:
We are already fetching particular commit for `e2e-node` tests.
https://github.com/ppc64le-cloud/k8s-ansible/blob/master/roles/k8s-node/tasks/main.yml#L22:#L32

@mkumatag Should we run Unit Tests also on ci/latest commit?

Reply:
yes for now, but in future - it will be helpful if we want to run tests against already releases branches etc..